tests/functional/api/groups/members_test.py

```diff
@@ -22,7 +27,7 @@ def test_it_returns_list_of_members_if_user_has_access_to_private_group(
         self, app, factories, db_session, group, user_with_token, token_auth_header
     ):
         user, _ = user_with_token
-        group.members.append(user)
+        group.memberships.append(GroupMembership(user=user))
```

Comment:
This can alternatively be written:

```
db_session.add(GroupMembership(user=user, group=group))
```

But it's slightly longer :man_shrugging: